sys/kern/subr_pcpu.c

```diff
@@ -343,9 +355,13 @@ DB_SHOW_COMMAND(dpcpu_off, db_show_dpcpu_off)
 	int id;
 
 	CPU_FOREACH(id) {
+#ifdef __CHERI_PURE_CAPABILITY__
+		db_printf("dpcpu_off[%2d] = %p)\n", id, (void *)dpcpu_off[id]);
+#else
 		db_printf("dpcpu_off[%2d] = 0x%jx (+ DPCPU_START = %p)\n",
```

Comment:
You can conditionally use this with the bias constant too (it's a bit of a redundant print for purecap but meh, reduces diffs a bit).

Reply:
Done.